backend/README.md

```diff
@@ -1,5 +1,20 @@
 # Backend Documentation
 
+## Box API Setup
+
+To start using Box, you must create a box account [here](https://account.box.com/login).
+
+After you create your new box account, you must visit the [Console](https://app.box.com/developers/console) to create your project.
+
+After creating your project, click into the project and go to *configuration*.
+
+In *configuration* tab, decide whether or not you want to authenticate using _OAuth_ or _JWT_. You can find more documentation about these two methods [here](https://developer.box.com/docs/quickstart-guides).
+
+Follow each step in the chosen authentication method, and you are all set!
+
+###box api document
+This [Box api document](https://developer.box.com/reference) has all the endpoints described for your own box integration.
+
```

Comment:
Also hit upon the json file that they might need in the backend as well as how our application currently handles Box (JWT vs OAuth, etc.) just so that Kiva has a good place of where to start looking if they were to setup or add onto the existing implementation.

Reply:
Also they'll probably need to integrate their own Box account so knowing what the app currently uses/how it handles it would be beneficial